root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js

```diff
@@ -143,21 +151,19 @@ function Fx_OnSubmit() {
     // ・ASP.NET Ajax Extension
     // のどちらも、通過する。
 
-    // ・IE6.0のhrefのdoPostBackだと動かない。
-    // ・OperaのhrefのdoPostBack（二重送信時限定）だと動かない。
-
-    // alert(navigator.appName);
-    // alert(navigator.appVersion);
-
     // ----------
 
+    // In Chrome, Safari and Firefox, document.readyState always returns complete
+    // Hence double transmission is prevented using Form_IsSubmitted flag
+
     // ↑Operaの場合、ポスト後直ちにタイマーが無効になるので、実質機能しない。
     //   ※ ただし、Ajaxの場合は、タイマーが無効にならない。
 
     // ----------
 
-    if (navigator.appName == "Microsoft Internet Explorer" || navigator.appVersion.indexOf("Trident") != -1) {
-
+    if (Browser_IsIE) {
+        
+        // Detected browser is Internet Explorer
         // スレイプニルもこちらに含まれる。
```

Comment:
@daisukenishino 

In case of using Sleipnir, **false** is substituted for variable "Brower_IsIE".
And **true** is substituted for variable "Browser_IsChrome".
So, this comment is not correct.